egs/tunisian_msa/s5/local/qcri_buckwalter2utf8.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+cut -d " " -f 1 qcri.txt > qcri_words_buckwalter.txt
+cut -d " " -f 2 qcri.txt > qcri_prons.txt
```

Comment:
Actually the option  -f 2 should be -f 2-. I realized this after I saw totally wrong decoding results...